tests/parse/test_parse_deleteriousness.py

```diff
@@ -0,0 +1,29 @@
+from scout.parse.variant.deleteriousness import parse_cadd
+
+
+def test_parse_cadd_info(cyvcf2_variant, case_obj):
+    """Test parsing a variant that has CADD score in the info field"""
+
+    variant = cyvcf2_variant
+    assert "CADD" not in variant.INFO
```

Comment:
Not wrong, but since you actually assign over it afterwards, not much use in checking either? Did you perhaps mean to double check that there are no other sources of CADD info on the variant? You do have control over that here as well, but..

Reply:
The idea was to simply test the parse_cadd function in scout.parse.variant.deleteriousness, not what's in variant.INFO when the cyvcf2_variant is created. To do that it is perfect that I assign the cadd in variant info, so I have control over the input and the output of the function

Reply:
Yes, but `variant.INFO["CADD"] = 23` is already overwriting whatever would have been found with `assert "CADD" not in variant.INFO`..

Reply:
It look as if you check for something else?

Reply:
It would make sense to check that the transcripts on that test variant have no `"cadd"`, because otherwise the test may not do what was intended..

Reply:
> Yes, but `variant.INFO["CADD"] = 23` is already overwriting whatever would have been found with `assert "CADD" not in variant.INFO`..

Ah now I get it, it looks like the test was meant to check something in variant.INFO because I wrote`assert "CADD" not in variant.INFO`. It's not, it was just to make sure I started from a "blank" state of variant.INFO. I'll remove that line, so it will be less confusing!

Reply:
> It would make sense to check that the transcripts on that test variant have no `"cadd"`, because otherwise the test may not do what was intended..

I'm not passing any transcript to the test, it's an empty list.